tvb_library/tvb/simulator/lab.py

```diff
@@ -46,9 +48,7 @@
 from tvb.basic.profile import TvbProfile
 TvbProfile.set_profile(TvbProfile.LIBRARY_PROFILE)
 
-from tvb.simulator.common import get_logger, log_debug
```

Comment:
I am thinking that leaving the import still here would be better, for now. to avoid breaking already working examples based on lab.py